src/components/FormGroup/FormGroup.vue

```diff
@@ -58,29 +59,28 @@ import {idProps} from '@/composables/useId';
 const props = defineProps({
     description: {
         type: String,
-        required: false,
         default: undefined,
     },
     helpText: {
         type: String,
-        required: false,
         default: undefined,
     },
     ...idProps,
     ...formControlProps,
     invalidFeedback: {
         type: String,
-        required: false,
         default: undefined,
     },
     label: {
         type: String,
-        required: false,
+        default: undefined,
+    },
+    labelClass: {
+        type: [String, Array, Object],
```

Comment:
I saw before couple times, can we use own custom type?

```suggestion
        type: Object as ClassType,
```

```js
type ClassType = PropType<[string, string[], Record<string, string>]>;
```

Reply:
This might be nice for TS but not for JS. Because it will not fail type validation when I pass a string.